packages/long-short-pair/src/utils/constants.ts

```diff
@@ -0,0 +1,9 @@
+import { BigDecimal, BigInt } from "@graphprotocol/graph-ts";
+import { toDecimal } from "./decimals";
+
+export const ZERO_ADDRESS = "0x0000000000000000000000000000000000000000";
```

Comment:
As mentioned before, might be useful to store LSP states here like [this](https://github.com/UMAprotocol/subgraphs/blob/master/packages/financial-contracts/src/utils/constants.ts#L12)

Reply:
yup. good idea. will refactor